app/views/pages/index.html.erb

```diff
@@ -0,0 +1,2 @@
+
+<%= markdown(File.read("app/views/pages/INDEX.markdown")) %>
```

Comment:
Can you add a new line after this? All files should end with a newline. Also no need for the empty line above this